marketplace/adoptium-marketplace-schema/src/main/java/net/adoptium/marketplace/schema/Vendor.java

```diff
@@ -0,0 +1,10 @@
+package net.adoptium.marketplace.schema;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+/*
+    Use a schema ref as adopt and adoptium will have a different subset of vendors
+*/
+public enum Vendor {
+    adoptium, adoptopenjdk, openjdk, alibaba, ibm, eclipse;
```

Comment:
is it better to have adopt and eclipse as aliases of adoptium or some such?

Reply:
can possibly get rid of eclipse, iirc I believe this is the former vendor used from openj9, however presumably the openj9 builds will not show in the market place, and semeru will have a vendor of ibm

Reply:
I'd possibly keep eclipse given the market confusion over eclipse vs adoptium vs temurin vs adopt.  Aliases will help here I think

Reply:
after last night's discussion, drop eclipse